sdk/eventhub/azure-eventhub/samples/async_samples/client_identity_authentication_async.py

```diff
@@ -60,6 +60,7 @@ async def run():
                     # New EventDataBatch object can be created here to send more data.
                     break
             await producer.send_batch(event_data_batch)
+    await credential.close()
```

Comment:
we actually have `async with credential:` on line 48, I think this `close` is not needed?

Reply:
you're right, I thought I was getting an error but I guess I wasn't 🤦